CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ Mapbox welcomes participation and contributions from everyone.
 #### Bug fixes and improvements
 - Now it's possible to see the same route in `NavigationRouteAlternativesObserver` but coming from an offboard router if the current route was built onboard which can be used to always prefer an offboard-generated route over onboard-generated one. This is a good practice because offboard-generated routes take live road conditions into account, have more precise ETAs, and can also be refreshed as the user drives and conditions change. Check `NavigationRouteAlternativesObserver` documentation for example usage. [#5768](https://github.com/mapbox/mapbox-navigation-android/pull/5768)
 - Improved behavior of enhanced location teleports on parallel roads, especially forks. [#5768](https://github.com/mapbox/mapbox-navigation-android/pull/5768)
+- Changed the internal vanishing route line feature implementation to use `line-trim-offset` property of `LineLayer` to improve performance of updates. [#5697](https://github.com/mapbox/mapbox-navigation-android/pull/5697)
```

Comment:
```suggestion
- Changed the internal vanishing route line feature implementation to use `line-trim-offset` property of `LineLayer` to improve performance of updates. The optimization is not available when `MapboxRouteLineOptions#styleInactiveRouteLegsIndependently` is enabled. [#5697](https://github.com/mapbox/mapbox-navigation-android/pull/5697)
```

Let's merge as is (to avoid additional CI cycle) but @tomaszrybakiewicz please refactor the message before committing the final release notes later today.